cmd/harbor/root/login.go

```diff
@@ -30,6 +31,11 @@ func LoginCommand() *cobra.Command {
 				serverAddress = args[0]
 			}
 
+			if Name == "" {
+				// Auto-generate the credential name if not provided
+				Name = generateCredentialName(serverAddress, Username)
+			}
+
```

Comment:
```suggestion
```

this is redundant please remove.

Reply:
@bupd I think it is essential while using with flags

Reply:
No @Althaf66 its not essential, Ultimately, everything is validated in the `runLogin` function. I tested it both with and without the flags.